apiClientDotNet/Clients/FirehoseClient.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using apiClientDotNet.Models;
+using apiClientDotNet.Models.Events;
+using apiClientDotNet.Utils;
+using apiClientDotNet.Listeners;
+using System.Net;
+using Newtonsoft.Json.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace apiClientDotNet
+{
+    public class FirehoseClient
```

Comment:
Hi @anthony-symphony, Firehose is a deprecated API that we don't want to promote anymore. Datahose will come very soon in replacement (with Agent 20.15). 

Could you remove this class from this PR?